hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/throttle/PressureAwareThroughputController.java

```diff
@@ -64,9 +64,9 @@
     }
   }
 
-  protected long maxThroughputUpperBound;
+  protected volatile long maxThroughputUpperBound;
```

Comment:
I think the old design is to recreate the controller so we do not need volatile here? Why change the behavior?